backend/dissemination/api/api_v1_0_3/create_functions.sql

```diff
@@ -13,48 +13,61 @@
 --
 -- To quote the work of Dav Pilkey, "remember this now."
 
-create or replace function getter(base text, item text) returns text 
+create or replace function api_v1_0_3.getter(base text, item text) returns text 
 as $getter$
 begin 
 	return current_setting(concat(base, '.', item), true);
 end;
 $getter$ language plpgsql;
 
-create or replace function get_jwt_claim(item text) returns text 
+create or replace function api_v1_0_3.get_jwt_claim(item text) returns text 
 as $get_jwt_claim$
 begin 
-	return getter('request.jwt.claim', item);
+	return api_v1_0_3.getter('request.jwt.claim', item);
 end;
 $get_jwt_claim$ language plpgsql;
 
-create or replace function get_header(item text) returns text
+create or replace function api_v1_0_3.get_header(item text) returns text
 as $get_header$
 begin 
-	raise info 'request.header % %', item, getter('request.header', item);
-	return getter('request.header', item);
+	-- raise info 'request.header % %', item, getter('request.header', item);
+	return api_v1_0_3.getter('request.header', item);
 end;
 $get_header$ LANGUAGE plpgsql;
 
 -- https://api-umbrella.readthedocs.io/en/latest/admin/api-backends/http-headers.html
--- I'd like to go to a model where we provide the API keys. 
--- However, for now, we're going to look for a role attached to an api.data.gov account.
--- These come in on `X-Api-Roles` as a comma-separated string.
-create or replace function has_tribal_data_access() returns boolean
+-- Previously, we were using a role attached to the key.
+-- This now uses an explicit table to determine if a key has tribal data access.
+-- What this means is that we must go through a commit/PR process in order to use
+-- keys that will have access to tribal data. This should be supported by a 
+-- ticketing process.
```

Comment:
It would be good to document the process now so it doesn't get lost.

Reply:
I'll add a ticket template.

Reply:
A ticket template has been added; comment updated.